gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,98 @@
+id: gigamon
+metric_id: gigamon
+backend_only: true
+
+facets:
+  - groups:
+      - Web Access
+    name: URL Path
+    path: http.url_details.path
+    source: log
+  - groups:
+      - Web Access
+    name: Status Code
+    path: http.status_code
+    source: log
+  - groups:
+      - DNS
+    name: DNS ID
+    path: dns.id
+    source: log
+  - groups:
+      - DNS
+    name: Response Code
+    path: dns.flags.rcode
+    source: log
+  - facetType: list
+    groups:
+      - gigamon
+    name: Application Name
+    path: app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_host
+    path: dns_host
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: dns_name
+    path: dns_name
+    source: log
+    type: string
+pipeline:
+  type: pipeline
+  name: gigamon
+  enabled: false
+  filter:
+    query: "source:gigamon"
+  processors:
+    - type: attribute-remapper
+      name: Map http_host` to `http.url_details.path`
+      enabled: true
+      sources:
+        - http_host
+      sourceType: attribute
+      target: http.url_details.path
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
```

Comment:
Can you replace this remapper with a category processor as seen [here](https://docs.datadoghq.com/logs/guide/remap-custom-severity-to-official-log-status/#web-access-logs)? Once remapped, we will need you to add a status remapper to remap the category to be the official status of the log.  

See section in the doc which will provide steps to add a category processor and a status remapper: 
[Web access logs](https://docs.datadoghq.com/logs/guide/remap-custom-severity-to-official-log-status/#web-access-logs)